Question on the file meson.build in nnstreamer root dir

# Issue Description
안녕하세요 
항상 감사하게 사용하고 있습니다

가장 상단에 있는 nnstreamer/meson.build 파일에 아래와 같은 부분이 있는데요

```
...
# Add redundant declaration flag when caffe2 and pytorch both are disabled
if not (pytorch_support_is_available and caffe2_support_is_available)
  redundant_decls_flag = '-Wredundant-decls'
  if cc.has_argument (redundant_decls_flag)
    #add_project_arguments([redundant_decls_flag], language: 'c')
  endif
  if cxx.has_argument (redundant_decls_flag)
    #add_project_arguments([redundant_decls_flag], language: 'cpp')
  endif
endif
...
```

주석만 읽어보면 caffe2와 pytorch가 둘다 disabled될 때 redundant error를 정의해주도록 의도하신 것 같은데, 조건문은 둘중 하나만 없어도 redundant error 정의해주는 것 같습니다 혹시 맞나요?
의도대로 동작중인 건지 궁금합니다
저는 caffe2 없이 libtorch만 넣고 자주 사용하는 데 해당 부분을 그냥 주석처리 하고 쓰다가 문득 궁금해져서 질문드립니다

읽어주셔서 감사합니다

Expected Result
============
caffe2 없이 libtorch만으로도 에러 없이 빌드

How to Reproduce
===============
1. libtorch 설치
2. caffe2 없이 nnstreamer 빌드 시도

Further Information
===============
None

주석을 해제하면 아래와 같은 redunant-decls 에러가 발생해요
```
/usr/lib/libtorch/include/c10/core/TensorOptions.h:130:13: error: redundant redeclaration of 'c10::DispatchKey c10::computeDispatchKey(c10::optional<c10::ScalarType>, c10::optional<c10::Layout>, c10::optional<c10::Device>)' in same scope [-Werror=redundant-decls]
 DispatchKey computeDispatchKey(
             ^~~~~~~~~~~~~~~~~~
/usr/lib/libtorch/include/c10/core/TensorOptions.h:22:13: note: previous declaration of 'c10::DispatchKey c10::computeDispatchKey(c10::optional<c10::ScalarType>, c10::optional<c10::Layout>, c10::optional<c10::Device>)'
 DispatchKey computeDispatchKey(
             ^~~~~~~~~~~~~~~~~~
In file included from /usr/lib/libtorch/include/ATen/TensorIndexing.h:11:0,
                 from /usr/lib/libtorch/include/ATen/ATen.h:20,
                 from /usr/lib/libtorch/include/torch/csrc/api/include/torch/types.h:3,
                 from /usr/lib/libtorch/include/torch/script.h:3,
                 from /usr/local/lib/nnstreamer/ext/nnstreamer/tensor_filter/tensor_filter_pytorch.cc:38:
/usr/lib/libtorch/include/ATen/NativeFunctions.h:850:22: error: redundant redeclaration of 'at::Tensor at::native::_sparse_mm(const at::Tensor&, const at::Tensor&)' in same scope -Werror=redundant-decls]
 TORCH_API at::Tensor _sparse_mm(const at::Tensor & sparse, const at::Tensor & dense);
                      ^~~~~~~~~~
/usr/lib/libtorch/include/ATen/NativeFunctions.h:847:22: note: previous declaration of 'at::Tensor at::native::_sparse_mm(const at::Tensor&, const at::Tensor&)'
 TORCH_API at::Tensor _sparse_mm(const at::Tensor & self, const at::Tensor & mat2);
                      ^~~~~~~~~~
In file included from /usr/lib/libtorch/include/ATen/TensorIndexing.h:11:0,
                 from /usr/lib/libtorch/include/ATen/ATen.h:20,
                 from /usr/lib/libtorch/include/torch/csrc/api/include/torch/types.h:3,
                 from /usr/lib/libtorch/include/torch/script.h:3,
                 from /usr/local/lib/nnstreamer/ext/nnstreamer/tensor_filter/tensor_filter_pytorch.cc:38:
/usr/lib/libtorch/include/ATen/NativeFunctions.h:1396:19: error: redundant redeclaration of 'int64_t at::native::sparse_dim_sparse(const at::Tensor&)' in same scope [-Werror=redundant-decls]
 TORCH_API int64_t sparse_dim_sparse(const at::Tensor & self);
                   ^~~~~~~~~~~~~~~~~
/usr/lib/libtorch/include/ATen/NativeFunctions.h:1395:19: note: previous declaration of 'int64_t at::native::sparse_dim_sparse(const at::Tensor&)'
 TORCH_API int64_t sparse_dim_sparse(const at::Tensor & self);
                   ^~~~~~~~~~~~~~~~~
/usr/lib/libtorch/include/ATen/NativeFunctions.h:1398:19: error: redundant redeclaration of 'int64_t at::native::dense_dim_sparse(const at::Tensor&)' in same scope [-Werror=redundant-decls]
 TORCH_API int64_t dense_dim_sparse(const at::Tensor & self);
                   ^~~~~~~~~~~~~~~~
/usr/lib/libtorch/include/ATen/NativeFunctions.h:1397:19: note: previous declaration of 'int64_t at::native::dense_dim_sparse(const at::Tensor&)'
 TORCH_API int64_t dense_dim_sparse(const at::Tensor & self);
                   ^~~~~~~~~~~~~~~~
/usr/lib/libtorch/include/ATen/NativeFunctions.h:2943:22: error: redundant redeclaration of 'at::Tensor at::native::_test_optional_intlist(const at::Tensor&, c10::optional<c10::ArrayRef<long int> >)' in same scope [-Werror=redundant-decls]
 TORCH_API at::Tensor _test_optional_intlist(const at::Tensor & values, c10::optional<at::IntArrayRef> addends);
                      ^~~~~~~~~~~~~~~~~~~~~~
/usr/lib/libtorch/include/ATen/NativeFunctions.h:2942:22: note: previous declaration of 'at::Tensor at::native::_test_optional_intlist(const at::Tensor&, c10::optional<c10::ArrayRef<long int> >)'
 TORCH_API at::Tensor _test_optional_intlist(const at::Tensor & values, c10::optional<at::IntArrayRef> addends);
                      ^~~~~~~~~~~~~~~~~~~~~~
In file included from /usr/lib/libtorch/include/ATen/core/List.h:480:0,
                 from /usr/lib/libtorch/include/ATen/TensorIndexing.h:13,
                 from /usr/lib/libtorch/include/ATen/ATen.h:20,
                 from /usr/lib/libtorch/include/torch/csrc/api/include/torch/types.h:3,
                 from /usr/lib/libtorch/include/torch/script.h:3,
                 from /usr/local/lib/nnstreamer/ext/nnstreamer/tensor_filter/tensor_filter_pytorch.cc:38:
/usr/lib/libtorch/include/ATen/core/List_inl.h:9:13: error: redundant redeclaration of 'std::string c10::toString(c10::TypePtr)' in same scope [-Werror=redundant-decls]
 std::string toString(TypePtr typePtr);
             ^~~~~~~~
In file included from /usr/lib/libtorch/include/ATen/core/Dict.h:392:0,
                 from /usr/lib/libtorch/include/ATen/core/ivalue_inl.h:7,
                 from /usr/lib/libtorch/include/ATen/core/ivalue.h:1177,
                 from /usr/lib/libtorch/include/ATen/core/List_inl.h:4,
                 from /usr/lib/libtorch/include/ATen/core/List.h:480,
                 from /usr/lib/libtorch/include/ATen/TensorIndexing.h:13,
                 from /usr/lib/libtorch/include/ATen/ATen.h:20,
                 from /usr/lib/libtorch/include/torch/csrc/api/include/torch/types.h:3,
                 from /usr/lib/libtorch/include/torch/script.h:3,
                 from /usr/local/lib/nnstreamer/ext/nnstreamer/tensor_filter/tensor_filter_pytorch.cc:38:
/usr/lib/libtorch/include/ATen/core/Dict_inl.h:20:13: note: previous declaration of 'std::string c10::toString(c10::TypePtr)'
 std::string toString(TypePtr typePtr);
             ^~~~~~~~
In file included from /usr/lib/libtorch/include/torch/custom_class.h:15:0,
                 from /usr/lib/libtorch/include/torch/library.h:856,
                 from /usr/lib/libtorch/include/torch/csrc/jit/runtime/operator.h:13,
                 from /usr/lib/libtorch/include/torch/csrc/jit/ir/ir.h:7,
                 from /usr/lib/libtorch/include/torch/csrc/jit/api/function_impl.h:4,
                 from /usr/lib/libtorch/include/torch/csrc/jit/api/method.h:7,
                 from /usr/lib/libtorch/include/torch/csrc/jit/api/object.h:6,
                 from /usr/lib/libtorch/include/torch/csrc/jit/frontend/tracer.h:9,
                 from /usr/lib/libtorch/include/torch/csrc/autograd/generated/variable_factories.h:12,
                 from /usr/lib/libtorch/include/torch/csrc/api/include/torch/types.h:7,
                 from /usr/lib/libtorch/include/torch/script.h:3,
                 from /usr/local/lib/nnstreamer/ext/nnstreamer/tensor_filter/tensor_filter_pytorch.cc:38:
/usr/lib/libtorch/include/torch/custom_class_detail.h:198:16: error: redundant redeclaration of 'bool torch::isCustomClass(const c10::IValue&)' in same scope [-Werror=redundant-decls]
 TORCH_API bool isCustomClass(const c10::IValue& v);
                ^~~~~~~~~~~~~
In file included from /usr/lib/libtorch/include/ATen/core/ivalue.h:1177:0,
                 from /usr/lib/libtorch/include/ATen/core/List_inl.h:4,
                 from /usr/lib/libtorch/include/ATen/core/List.h:480,
                 from /usr/lib/libtorch/include/ATen/TensorIndexing.h:13,
                 from /usr/lib/libtorch/include/ATen/ATen.h:20,
                 from /usr/lib/libtorch/include/torch/csrc/api/include/torch/types.h:3,
                 from /usr/lib/libtorch/include/torch/script.h:3,
                 from /usr/local/lib/nnstreamer/ext/nnstreamer/tensor_filter/tensor_filter_pytorch.cc:38:
/usr/lib/libtorch/include/ATen/core/ivalue_inl.h:29:16: note: previous declaration of 'bool torch::isCustomClass(const c10::IValue&)'
 TORCH_API bool isCustomClass(const c10::IValue& v);
                ^~~~~~~~~~~~~
In file included from /usr/lib/libtorch/include/torch/csrc/jit/runtime/graph_executor.h:9:0,
                 from /usr/lib/libtorch/include/torch/csrc/jit/api/function_impl.h:5,
                 from /usr/lib/libtorch/include/torch/csrc/jit/api/method.h:7,
                 from /usr/lib/libtorch/include/torch/csrc/jit/api/object.h:6,
                 from /usr/lib/libtorch/include/torch/csrc/jit/frontend/tracer.h:9,
                 from /usr/lib/libtorch/include/torch/csrc/autograd/generated/variable_factories.h:12,
                 from /usr/lib/libtorch/include/torch/csrc/api/include/torch/types.h:7,
                 from /usr/lib/libtorch/include/torch/script.h:3,
                 from /usr/local/lib/nnstreamer/ext/nnstreamer/tensor_filter/tensor_filter_pytorch.cc:38:
/usr/lib/libtorch/include/torch/csrc/jit/runtime/interpreter.h:17:16: error: redundant redeclaration of 'void at::launch(std::function<void()>)' in same scope [-Werror=redundant-decls]
 TORCH_API void launch(std::function<void()> func);
                ^~~~~~
In file included from /usr/lib/libtorch/include/torch/csrc/jit/api/method.h:3:0,
                 from /usr/lib/libtorch/include/torch/csrc/jit/api/object.h:6,
                 from /usr/lib/libtorch/include/torch/csrc/jit/frontend/tracer.h:9,
                 from /usr/lib/libtorch/include/torch/csrc/autograd/generated/variable_factories.h:12,
                 from /usr/lib/libtorch/include/torch/csrc/api/include/torch/types.h:7,
                 from /usr/lib/libtorch/include/torch/script.h:3,
                 from /usr/local/lib/nnstreamer/ext/nnstreamer/tensor_filter/tensor_filter_pytorch.cc:38:
/usr/lib/libtorch/include/ATen/core/function.h:12:16: note: previous declaration of 'void at::launch(std::function<void()>)'
 TORCH_API void launch(std::function<void()> func);
                ^~~~~~
cc1plus: all warnings being treated as errors
[210/219] Compiling C++ object 'ext/nnstreamer/tensor_filter/584bea7@@nnstreamer_filter_pytorch@sta/tensor_filter_pytorch.cc.o'.
FAILED: ext/nnstreamer/tensor_filter/584bea7@@nnstreamer_filter_pytorch@sta/tensor_filter_pytorch.cc.o 
c++ -Iext/nnstreamer/tensor_filter/584bea7@@nnstreamer_filter_pytorch@sta -Iext/nnstreamer/tensor_filter -I../ext/nnstreamer/tensor_filter -Igst/nnstreamer -I../gst/nnstreamer -Igst/nnstreamer/./include/ -I../gst/nnstreamer/./include/ -I/usr/lib/libtorch/include -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -fdiagnostics-color=always -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wnon-virtual-dtor -Wextra -Werror -std=c++14 -g '-DVERSION="2.1.0"' '-DVERSION_MAJOR="2"' '-DVERSION_MINOR="1"' '-DVERSION_MICRO="0"' -Wmissing-braces -Wmaybe-uninitialized -Wwrite-strings -Wformat -Wformat-nonliteral -Wformat-security -Winit-self -Waddress -Wno-multichar -Wno-redundant-decls -Wno-unused-parameter -D_GLIBCXX_USE_CXX11_ABI=0 -Wvla -Wpointer-arith -DENABLE_PYTORCH=1 -DHAVE_ORC=1 -Wredundant-decls -fPIC -DPYTORCH_VER_ATLEAST_1_2_0=1 -pthread -MD -MQ 'ext/nnstreamer/tensor_filter/584bea7@@nnstreamer_filter_pytorch@sta/tensor_filter_pytorch.cc.o' -MF 'ext/nnstreamer/tensor_filter/584bea7@@nnstreamer_filter_pytorch@sta/tensor_filter_pytorch.cc.o.d' -o 'ext/nnstreamer/tensor_filter/584bea7@@nnstreamer_filter_pytorch@sta/tensor_filter_pytorch.cc.o' -c /usr/local/lib/nnstreamer/ext/nnstreamer/tensor_filter/tensor_filter_pytorch.cc
In file included from /usr/lib/libtorch/include/ATen/Operators.h:14:0,
                 from /usr/lib/libtorch/include/ATen/core/TensorBody.h:3,
                 from /usr/lib/libtorch/include/ATen/Tensor.h:3,
                 from /usr/lib/libtorch/include/ATen/Context.h:4,
                 from /usr/lib/libtorch/include/ATen/ATen.h:9,
                 from /usr/lib/libtorch/include/torch/csrc/api/include/torch/types.h:3,
                 from /usr/lib/libtorch/include/torch/script.h:3,
                 from /usr/local/lib/nnstreamer/ext/nnstreamer/tensor_filter/tensor_filter_pytorch.cc:38:
/usr/lib/libtorch/include/c10/core/TensorOptions.h:130:13: error: redundant redeclaration of 'c10::DispatchKey c10::computeDispatchKey(c10::optional<c10::ScalarType>, c10::optional<c10::Layout>, c10::optional<c10::Device>)' in same scope [-Werror=redundant-decls]
 DispatchKey computeDispatchKey(
             ^~~~~~~~~~~~~~~~~~
/usr/lib/libtorch/include/c10/core/TensorOptions.h:22:13: note: previous declaration of 'c10::DispatchKey c10::computeDispatchKey(c10::optional<c10::ScalarType>, c10::optional<c10::Layout>, c10::optional<c10::Device>)'
 DispatchKey computeDispatchKey(
             ^~~~~~~~~~~~~~~~~~
In file included from /usr/lib/libtorch/include/ATen/TensorIndexing.h:11:0,
                 from /usr/lib/libtorch/include/ATen/ATen.h:20,
                 from /usr/lib/libtorch/include/torch/csrc/api/include/torch/types.h:3,
                 from /usr/lib/libtorch/include/torch/script.h:3,
                 from /usr/local/lib/nnstreamer/ext/nnstreamer/tensor_filter/tensor_filter_pytorch.cc:38:
/usr/lib/libtorch/include/ATen/NativeFunctions.h:850:22: error: redundant redeclaration of 'at::Tensor at::native::_sparse_mm(const at::Tensor&, const at::Tensor&)' in same scope -Werror=redundant-decls]
 TORCH_API at::Tensor _sparse_mm(const at::Tensor & sparse, const at::Tensor & dense);
                      ^~~~~~~~~~
/usr/lib/libtorch/include/ATen/NativeFunctions.h:847:22: note: previous declaration of 'at::Tensor at::native::_sparse_mm(const at::Tensor&, const at::Tensor&)'
 TORCH_API at::Tensor _sparse_mm(const at::Tensor & self, const at::Tensor & mat2);
                      ^~~~~~~~~~
In file included from /usr/lib/libtorch/include/ATen/TensorIndexing.h:11:0,
                 from /usr/lib/libtorch/include/ATen/ATen.h:20,
                 from /usr/lib/libtorch/include/torch/csrc/api/include/torch/types.h:3,
                 from /usr/lib/libtorch/include/torch/script.h:3,
                 from /usr/local/lib/nnstreamer/ext/nnstreamer/tensor_filter/tensor_filter_pytorch.cc:38:
/usr/lib/libtorch/include/ATen/NativeFunctions.h:1396:19: error: redundant redeclaration of 'int64_t at::native::sparse_dim_sparse(const at::Tensor&)' in same scope [-Werror=redundant-decls]
 TORCH_API int64_t sparse_dim_sparse(const at::Tensor & self);
                   ^~~~~~~~~~~~~~~~~
/usr/lib/libtorch/include/ATen/NativeFunctions.h:1395:19: note: previous declaration of 'int64_t at::native::sparse_dim_sparse(const at::Tensor&)'
 TORCH_API int64_t sparse_dim_sparse(const at::Tensor & self);
                   ^~~~~~~~~~~~~~~~~
/usr/lib/libtorch/include/ATen/NativeFunctions.h:1398:19: error: redundant redeclaration of 'int64_t at::native::dense_dim_sparse(const at::Tensor&)' in same scope [-Werror=redundant-decls]
 TORCH_API int64_t dense_dim_sparse(const at::Tensor & self);
                   ^~~~~~~~~~~~~~~~
/usr/lib/libtorch/include/ATen/NativeFunctions.h:1397:19: note: previous declaration of 'int64_t at::native::dense_dim_sparse(const at::Tensor&)'
 TORCH_API int64_t dense_dim_sparse(const at::Tensor & self);
                   ^~~~~~~~~~~~~~~~
/usr/lib/libtorch/include/ATen/NativeFunctions.h:2943:22: error: redundant redeclaration of 'at::Tensor at::native::_test_optional_intlist(const at::Tensor&, c10::optional<c10::ArrayRef<long int> >)' in same scope [-Werror=redundant-decls]
 TORCH_API at::Tensor _test_optional_intlist(const at::Tensor & values, c10::optional<at::IntArrayRef> addends);
                      ^~~~~~~~~~~~~~~~~~~~~~
/usr/lib/libtorch/include/ATen/NativeFunctions.h:2942:22: note: previous declaration of 'at::Tensor at::native::_test_optional_intlist(const at::Tensor&, c10::optional<c10::ArrayRef<long int> >)'
 TORCH_API at::Tensor _test_optional_intlist(const at::Tensor & values, c10::optional<at::IntArrayRef> addends);
                      ^~~~~~~~~~~~~~~~~~~~~~
In file included from /usr/lib/libtorch/include/ATen/core/List.h:480:0,
                 from /usr/lib/libtorch/include/ATen/TensorIndexing.h:13,
                 from /usr/lib/libtorch/include/ATen/ATen.h:20,
                 from /usr/lib/libtorch/include/torch/csrc/api/include/torch/types.h:3,
                 from /usr/lib/libtorch/include/torch/script.h:3,
                 from /usr/local/lib/nnstreamer/ext/nnstreamer/tensor_filter/tensor_filter_pytorch.cc:38:
/usr/lib/libtorch/include/ATen/core/List_inl.h:9:13: error: redundant redeclaration of 'std::string c10::toString(c10::TypePtr)' in same scope [-Werror=redundant-decls]
 std::string toString(TypePtr typePtr);
             ^~~~~~~~
In file included from /usr/lib/libtorch/include/ATen/core/Dict.h:392:0,
                 from /usr/lib/libtorch/include/ATen/core/ivalue_inl.h:7,
                 from /usr/lib/libtorch/include/ATen/core/ivalue.h:1177,
                 from /usr/lib/libtorch/include/ATen/core/List_inl.h:4,
                 from /usr/lib/libtorch/include/ATen/core/List.h:480,
                 from /usr/lib/libtorch/include/ATen/TensorIndexing.h:13,
                 from /usr/lib/libtorch/include/ATen/ATen.h:20,
                 from /usr/lib/libtorch/include/torch/csrc/api/include/torch/types.h:3,
                 from /usr/lib/libtorch/include/torch/script.h:3,
                 from /usr/local/lib/nnstreamer/ext/nnstreamer/tensor_filter/tensor_filter_pytorch.cc:38:
/usr/lib/libtorch/include/ATen/core/Dict_inl.h:20:13: note: previous declaration of 'std::string c10::toString(c10::TypePtr)'
 std::string toString(TypePtr typePtr);
             ^~~~~~~~
In file included from /usr/lib/libtorch/include/torch/custom_class.h:15:0,
                 from /usr/lib/libtorch/include/torch/library.h:856,
                 from /usr/lib/libtorch/include/torch/csrc/jit/runtime/operator.h:13,
                 from /usr/lib/libtorch/include/torch/csrc/jit/ir/ir.h:7,
                 from /usr/lib/libtorch/include/torch/csrc/jit/api/function_impl.h:4,
                 from /usr/lib/libtorch/include/torch/csrc/jit/api/method.h:7,
                 from /usr/lib/libtorch/include/torch/csrc/jit/api/object.h:6,
                 from /usr/lib/libtorch/include/torch/csrc/jit/frontend/tracer.h:9,
                 from /usr/lib/libtorch/include/torch/csrc/autograd/generated/variable_factories.h:12,
                 from /usr/lib/libtorch/include/torch/csrc/api/include/torch/types.h:7,
                 from /usr/lib/libtorch/include/torch/script.h:3,
                 from /usr/local/lib/nnstreamer/ext/nnstreamer/tensor_filter/tensor_filter_pytorch.cc:38:
/usr/lib/libtorch/include/torch/custom_class_detail.h:198:16: error: redundant redeclaration of 'bool torch::isCustomClass(const c10::IValue&)' in same scope [-Werror=redundant-decls]
 TORCH_API bool isCustomClass(const c10::IValue& v);
                ^~~~~~~~~~~~~
In file included from /usr/lib/libtorch/include/ATen/core/ivalue.h:1177:0,
                 from /usr/lib/libtorch/include/ATen/core/List_inl.h:4,
                 from /usr/lib/libtorch/include/ATen/core/List.h:480,
                 from /usr/lib/libtorch/include/ATen/TensorIndexing.h:13,
                 from /usr/lib/libtorch/include/ATen/ATen.h:20,
                 from /usr/lib/libtorch/include/torch/csrc/api/include/torch/types.h:3,
                 from /usr/lib/libtorch/include/torch/script.h:3,
                 from /usr/local/lib/nnstreamer/ext/nnstreamer/tensor_filter/tensor_filter_pytorch.cc:38:
/usr/lib/libtorch/include/ATen/core/ivalue_inl.h:29:16: note: previous declaration of 'bool torch::isCustomClass(const c10::IValue&)'
 TORCH_API bool isCustomClass(const c10::IValue& v);
                ^~~~~~~~~~~~~
In file included from /usr/lib/libtorch/include/torch/csrc/jit/runtime/graph_executor.h:9:0,
                 from /usr/lib/libtorch/include/torch/csrc/jit/api/function_impl.h:5,
                 from /usr/lib/libtorch/include/torch/csrc/jit/api/method.h:7,
                 from /usr/lib/libtorch/include/torch/csrc/jit/api/object.h:6,
                 from /usr/lib/libtorch/include/torch/csrc/jit/frontend/tracer.h:9,
                 from /usr/lib/libtorch/include/torch/csrc/autograd/generated/variable_factories.h:12,
                 from /usr/lib/libtorch/include/torch/csrc/api/include/torch/types.h:7,
                 from /usr/lib/libtorch/include/torch/script.h:3,
                 from /usr/local/lib/nnstreamer/ext/nnstreamer/tensor_filter/tensor_filter_pytorch.cc:38:
/usr/lib/libtorch/include/torch/csrc/jit/runtime/interpreter.h:17:16: error: redundant redeclaration of 'void at::launch(std::function<void()>)' in same scope [-Werror=redundant-decls]
 TORCH_API void launch(std::function<void()> func);
                ^~~~~~
In file included from /usr/lib/libtorch/include/torch/csrc/jit/api/method.h:3:0,
                 from /usr/lib/libtorch/include/torch/csrc/jit/api/object.h:6,
                 from /usr/lib/libtorch/include/torch/csrc/jit/frontend/tracer.h:9,
                 from /usr/lib/libtorch/include/torch/csrc/autograd/generated/variable_factories.h:12,
                 from /usr/lib/libtorch/include/torch/csrc/api/include/torch/types.h:7,
                 from /usr/lib/libtorch/include/torch/script.h:3,
                 from /usr/local/lib/nnstreamer/ext/nnstreamer/tensor_filter/tensor_filter_pytorch.cc:38:
/usr/lib/libtorch/include/ATen/core/function.h:12:16: note: previous declaration of 'void at::launch(std::function<void()>)'
 TORCH_API void launch(std::function<void()> func);

```

--- a/meson.build
+++ b/meson.build
@@ -456,7 +456,7 @@ foreach name, value: project_args
 endforeach
 
 # Add redundant declaration flag when caffe2 and pytorch both are disabled
-if not (pytorch_support_is_available and caffe2_support_is_available)
+if not (pytorch_support_is_available or caffe2_support_is_available)
   redundant_decls_flag = '-Wredundant-decls'
   if cc.has_argument (redundant_decls_flag)
     add_project_arguments([redundant_decls_flag], language: 'c')

